lib/transaction/priority_queue.go

```diff
@@ -80,14 +81,16 @@ type PriorityQueue struct {
 	pq        priorityQueue
 	currOrder uint64
 	txs       map[common.Hash]*Item
+	sleepTime time.Duration
 	sync.Mutex
 }
 
 // NewPriorityQueue creates new instance of PriorityQueue
 func NewPriorityQueue() *PriorityQueue {
 	spq := &PriorityQueue{
-		pq:  make(priorityQueue, 0),
-		txs: make(map[common.Hash]*Item),
+		pq:        make(priorityQueue, 0),
+		txs:       make(map[common.Hash]*Item),
+		sleepTime: time.Millisecond * 10,
```

Comment:
supernit - unit after factor

```suggestion
		sleepTime: 10 * time.Millisecond,
```